src/planwise/client/projects2/handlers.cljs

```diff
@@ -31,6 +39,23 @@
                      (assoc :list new-list))
       :navigate  (routes/projects2-show {:id project-id})})))
 
+(rf/reg-event-fx
+ :projects2/infer-step
+ in-projects2
+ (fn [{:keys [db]} [_ project]]
+   (let [steps ["goal", "consumers", "providers", "coverage", "actions", "review"]
```

Comment:
You can use the sections data structure here again and build a function which returns the first item for which the spec fails.